app/src/test/java/com/mstudio/superheroarch/presentation/mapper/CharacterDataMapperTest.kt

```diff
@@ -0,0 +1,31 @@
+package com.mstudio.superheroarch.presentation.mapper
+
+import com.mstudio.superheroarch.RickAndMortyRepositoryInstruments
+import com.mstudio.superheroarch.presentation.model.toCharacterAndEpisode
+import com.mstudio.superheroarch.remotedatasource.model.toEpisode
+import com.mstudio.superheroarch.remotedatasource.model.toTheMovieDbEpisode
+import org.junit.Assert.assertEquals
+import org.junit.Test
+
+class CharacterDataMapperTest {
+
+    @Test
+    fun `given character data, when mapped to character and episode data, then return correct data`() {
+        val characterData = RickAndMortyRepositoryInstruments.givenCharacterData()
+        val episodeData = RickAndMortyRepositoryInstruments.givenAnEpisodeRemoteEntity().toEpisode()
+        val episodeExtraData = RickAndMortyRepositoryInstruments.givenAnEpisodeExtraData().toTheMovieDbEpisode()
+        val characterAndEpisodeData = characterData.toCharacterAndEpisode(episodeData, episodeExtraData)
```

Comment:
this naming makes difficult to understand the test, better to use expected and actual